src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
It is a good and common practice to put the styles for the body of the document in a separate style file. It allows you to better organize your project and divide styles into logical blocks. 
Since this is more of a file for resetting the default styles, I would recommend moving the styles for the body to another file, such as general styles.